include/ppx/application.h

```diff
@@ -326,12 +328,19 @@ class Application
     {
         return mWindow;
     }
-    grfx::InstancePtr  GetInstance() const { return mInstance; }
-    grfx::DevicePtr    GetDevice() const { return mDevice; }
-    grfx::QueuePtr     GetGraphicsQueue(uint32_t index = 0) const { return GetDevice()->GetGraphicsQueue(index); }
-    grfx::QueuePtr     GetComputeQueue(uint32_t index = 0) const { return GetDevice()->GetComputeQueue(index); }
-    grfx::QueuePtr     GetTransferQueue(uint32_t index = 0) const { return GetDevice()->GetTransferQueue(index); }
-    grfx::SwapchainPtr GetSwapchain() const { return mSwapchain; }
+    grfx::InstancePtr GetInstance() const { return mInstance; }
+    grfx::DevicePtr   GetDevice() const { return mDevice; }
+    grfx::QueuePtr    GetGraphicsQueue(uint32_t index = 0) const { return GetDevice()->GetGraphicsQueue(index); }
+    grfx::QueuePtr    GetComputeQueue(uint32_t index = 0) const { return GetDevice()->GetComputeQueue(index); }
+    grfx::QueuePtr    GetTransferQueue(uint32_t index = 0) const { return GetDevice()->GetTransferQueue(index); }
+
+    // index here is for XR applications to fetch the swapchain of different views
```

Comment:
nit: Comment formatting - capitalized and periods at end. (https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar)

Reply:
fixed